### PR TITLE
longevity syndrome tweaks

### PR DESCRIPTION
**Pull request by MadmanMartian**
Gives a bit of feedback to longevity syndrome

Can't say for sure if switching from ^= bit to &= ~bit is any different, but it's consistent with the rest of the codebase.

Also, why was the feedback of the deactivation backlash a notice and not a warning?

Sorry for messing with virology stuff during the rework

:cl:
 * tweak: The longevity symptom now backlash-ages you according to the multiplier, rather than you gaining an arbitrary 8 years of age. Age currently does not affect anything ingame.

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
Changelog required now that multiplier adds to age

File: code/modules/virus2/effect/stage_4.dm [hunk 1]
```diff
@@ -166,15 +166,16 @@
 		var/mob/living/carbon/human/H = mob
 		for (var/datum/organ/external/E in H.organs)
 			if (E.status & ORGAN_BROKEN && prob(30))
-				E.status ^= ORGAN_BROKEN
+				to_chat(user, "<span class = 'notice'>You feel the bones in your [E.display_name] set back into place.</span>")
```
Comment (boy2mantwicethefam) [inline code/modules/virus2/effect/stage_4.dm]: H, not user

Comment (MadmanMartian) [reply code/modules/virus2/effect/stage_4.dm]: good spot, I'm sleepycoding so out of habit put user.

**Comment by MadmanMartian** [comment]
Will add changelog, but age doesn't seem to matter at all.

**Pull request merged**